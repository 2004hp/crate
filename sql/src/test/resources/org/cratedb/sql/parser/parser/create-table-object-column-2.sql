CREATE TABLE t2(col1 object(STRICT) as (col11 integer, col12 string))