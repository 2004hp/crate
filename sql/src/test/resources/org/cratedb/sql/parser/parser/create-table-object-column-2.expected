org.cratedb.sql.parser.parser.CreateTableNode@7bfc07a7
name: t2
statementType: CREATE TABLE
lockGranularity: R
existenceCheck: NO_CONDITION
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@37e80da0

	[0]:	
	org.cratedb.sql.parser.parser.ObjectColumnDefinitionNode@63d6a00c
	type: OBJECT
	name: col1
	elementType: null
	objectType: STRICT
	subColumns: 		
		org.cratedb.sql.parser.parser.TableElementList@1fa5a709

		[0]:		
		org.cratedb.sql.parser.parser.ColumnDefinitionNode@3479501a
		type: INTEGER
		name: col11
		elementType: null
		[1]:		
		org.cratedb.sql.parser.parser.ColumnDefinitionNode@69c7f04d
		type: string
		name: col12
		elementType: null
